dbt/macros/assessment_progress_pin_count.sql

```diff
@@ -1,4 +1,4 @@
--- Macros that counts the number of the number of PINs with values and the total
+-- Macro that counts the number of the number of PINs with values and the total
```

Comment:
This was just a grammar error I found while poking around the macros. Not related to this PR.